### PR TITLE
Make nextjs starter kit tests more useful. Solve some SDK typescript problems.

### PR DESCRIPTION
**Pull request by kevinstubbs**

### CODE REVIEW

File: starters/gatsby-starter/src/components/seo.jsx [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import React from 'react';
```
Comment (Aumkar) [inline starters/gatsby-starter/src/components/seo.jsx:1]: Can we remove this import?

**Review by aumkar — COMMENTED** [review]
LGTM! Just one minor comment

File: starters/gatsby-starter/src/components/seo.jsx (hunk 1 above)
Comment (kevinstubbs) [reply starters/gatsby-starter/src/components/seo.jsx:1]: Importing React like this helps the compiler on .jsx and .tsx files even if we aren't directly using it. I'm not sure in what circumstances it's actually necessary. In this case, I added it in order to fix a gatsby build error.
@Aumkar

Comment (Aumkar) [reply starters/gatsby-starter/src/components/seo.jsx:1]: Ok, makes sense

**Review by aumkar — APPROVED** [review]

**Pull request merged**